<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="dsbScaleX">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>period</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="10">
    <widget class="PlotSceneWidget" name="graphicsPlot"/>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of Points:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>570</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Precision:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QSpinBox" name="sbBuffer">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="sbPrecision">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QSpinBox" name="sbPeriod">
     <property name="suffix">
      <string>ms</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>700</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="dsbScaleY">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Scale (dx,dy):</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotSceneWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">plotscenewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
